### ISSUE TITLE
Proper formatting

### ISSUE THREAD

**Issue by bontchev** [huuck/ADBHoney#1]
If it wouldn't be too much of a trouble, could you please spend some time to format the program properly?

Python is very peculiar about line indentation, since it determines the beginning and ending of blocks. Please indent each block by exactly 4 spaces. As it currently is, different lines have different indentation, the indentation itself is a mixture of tabs and spaces and it is very difficult to understand where the block begins and where it ends.

For instance, where does the `while True:` loop in function `process_connection()` actually end? The indentation suggests that it ends after the second `try:...except` block - but that can't be the case, because there are `continue` operators afterwards with no corresponding loop.

There are other oddities in the code, like, it seems to me that the variable `filename` is sometimes used without being initialized - but maybe I don't understand something because the weird line indentations have confused me about the structure of the program...

**Comment by huuck** [comment]
Gonna do a full lint next week probably.

And yeah, filename was glued there in a rush as the name of the file is not always sent using the packet sequence.

**Comment by bontchev** [comment]
Please try to fix the indentation first. This should be very easy for you and it will allow others to better understand the code and make improvements or bug fixes to it.

**Comment by fe7ch** [comment]
@huuck I've made a PR with fixed indentation & some other basic fixes.

**Comment by fe7ch** [comment]
Furthermore, there was a crucial problem that the call to conn.close() was unreachable.

**Comment by huuck** [comment]
@fe7ch thanks a lot mate! Try to fix the filename issue and I'll merge it ASAP

**Comment by fe7ch** [comment]
@huuck I've already did it. I initialize it with "unknown" to avoid crash with referencing to undefined variable. Later, if we know the real filename, the variable will be set to this name.

**Comment by huuck** [comment]
![image](https://user-images.githubusercontent.com/3353285/48943026-7aa37e00-ef2a-11e8-8357-6ecbac4cfe65.png)
It gets reset to unknown after each read packet :P

**Comment by fe7ch** [comment]
Oups :D Moved the initialization of the variable out of the loop for now.

**Comment by huuck** [comment]
@bontchev Any more suggestions or we can close this? Looks like @fe7ch did an awesome job in fixing a lot of this stuff.

**Comment by ghost** [comment]
Yes, one. Disable download of binaries per command switch so we don't get flagged by ISP's. Although I am not sure yet if the download (sinkhole?) triggers it, or the Honeypot just listening.

**Comment by fe7ch** [comment]
>Disable download of binaries per command switch so we don't get flagged by ISP's

ADHoney doesn't download anything on its own for now. The "downloaded" files refers to the files that was uploaded to the honeypot via adb. So you should be safe at least for now ;)

**Comment by bontchev** [comment]
@fe7ch, thanks a lot for fixing the indentation. Could you please take a look at the other file (`protocol.py`) too? There are a few minor indentation issues that have to be fixed there too; they are much easier.

@HermanusF, it would be preferable to have a separate issue about this. I've been thinking about this subject too. As @fe7ch said, currently the honeypot does not initiate a download (e.g., when the attacker issues a `wget` shell command. I think that it should, and you make a good point that it should be optional. In addition, currently the honeypot writes uploaded file to disk every time. This should be done only if the file doesn't already exist. But these should be discussed in a separate issue. Let's keep this one about formatting only.

**Comment by fe7ch** [comment]
>Could you please take a look at the other file (protocol.py) too?

I'll take a look on Monday.

**Comment by ghost** [comment]
My provider was triggered by my connection having port 5555 open, so catching raw files isn't the "problem" at all, they told me 5 minutes ok.  So you can forget my request for me, but I think it's a 'nice to have' feature anyways.

**Comment by huuck** [comment]
Allright then ^^ The downloaded files have no impact anyhow (low interaction and all that), so it's safe to save them to disk.

**Comment by ghost** [comment]
Correct.

**Comment by bontchev** [comment]
OK, I have fixed `protocol.py` - as I said, it was much easier. Submitted pull request #19. Once it is merged, I'll close this issue.
